main/api/src/controllers/status/index.ts

```diff
@@ -1,9 +1,8 @@
 import { StatusCodes } from "http-status-codes";
 
 import { BasicControllerType } from "../../types";
+import { config } from "../../config";
 
 export const getStatus: BasicControllerType = (req, res) => {
-  res.status(StatusCodes.OK).send({
-    status: "OK",
-  });
+  res.status(StatusCodes.OK).send(config.version);
```

Comment:
Keep `status: "OK"` and add version to the response. Sorry if the requirements weren't clear.

Reply:
Also update the test for that controller to include the version.